pasqal_cloud/_version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.3.3"
+__version__ = "0.3.4"
```

Comment:
I don't think this warrants a version bump tbh

Reply:
Alright, I had a doubt regarding the python version dependency